test/hierarchical-data.test.js

```diff
@@ -15,7 +15,9 @@ describe('cap/samples - Hierarchical Data', ()=>{
 	const {expect} = cds.test
 
 	before ('bootstrap sqlite in-memory db...', async()=>{
-		await cds.deploy (csn) .to ('sqlite::memory:') // REVISIT: cds.compile.to.sql should accept cds.compiled.for.nodejs models
+		// REVISIT: cds.compile.to.sql should accept cds.compiled.for.nodejs models
+		// REVISIT: how to better configure new sqlite
+		await cds.deploy (csn) .to ({impl: '@cap-js/sqlite'}) // 'sqlite::memory:'
```

Comment:
@danjoa @johannes-vogel this is needed today to make `cds.deploy` use the new sqlite service
A package.json with a dependency next to the test doesn't help, and is maybe not the best way either.
So: how shall such a test indicate it wants to use new sqlite.

Reply:
Good question as only the plugin configures itself as the db and that's not run here.